tests/test_deblend.py

```diff
@@ -81,6 +86,28 @@ def setUp(self):
         for b, coadd in enumerate(self.coadds):
             coadd.setPsf(psfs[b])
 
+        # Initialize a Butler
+        self.root = makeTestTempDir(TESTDIR)
```

Comment:
I think you can use `makeTestRepo` to simplify things a bit.

Reply:
I'm not sure how to use this, as I can't figure out where to pass the `run` and it still seems to require creating a temporary root directory and creating an empty Config object. Is there an easier way that I'm missing?

Reply:
Yes. Use pipe base InMemoryDatasetHandle.

Reply:
Look in `pipe_tasks/tests/test_getRegionTimeFromVisit.py` for an example. The run can be given to butler.get. You don't need the config because it returns the butler directly.